mlflow/llama_index/tracer.py

```diff
@@ -36,6 +36,44 @@
 _logger = logging.getLogger(__name__)
 
 
+def set_llama_index_tracer():
+    """
+    Set the MlflowSpanHandler and MlflowEventHandler to the global dispatcher.
+    If the handlers are already set, skip setting.
+    """
+    from llama_index.core.instrumentation import get_dispatcher
+
+    dsp = get_dispatcher()
+
+    span_handler = None
+    for handler in dsp.span_handlers:
+        if handler.class_name() == "MlflowSpanHandler":
```

Comment:
Is the class name check here solely to avoid a local scoped import to avoid a circular dep reference?

Reply:
oh it doesn't have to be name check, will replace with `isinstance`